src/encoded/types/cart.py

```diff
@@ -96,3 +101,51 @@ def get_or_create_cart_by_user(context, request):
         '@type': ['result'],
         '@graph': carts or [cart]
     }
+
+
+@view_config(context=Cart.Collection, request_method='PUT', permission='save-carts', name='put-cart')
+def create_cart_by_user(context, request):
+    userid = get_userid(request)
+    user = _get_user(request, userid)
+    blocked_statuses = ['deleted'] if 'group.admin' not in request.effective_principals else []
+    carts = get_cart_objects_by_user(request, userid, blocked_statuses)
+    cart_atids = [cart['@id'] for cart in carts]
+    cart_status = request.json.get('status')
+    cart_name = request.json.get('name', '').strip()
+    # User writing a new cart; check for cart overflow and naming conflicts
+    countable_carts = []
+    if cart_status != 'disabled':
+        countable_carts = [
+            cart
+            for cart in carts
+            if cart['status'] not in ['disabled', 'deleted']
+        ]
+    if len(countable_carts) >= CART_USER_MAX:
+        msg = 'Users cannot have more than {} carts'.format(CART_USER_MAX)
+        raise HTTPBadRequest(explanation=msg)
+    name_conflict = any(
+        [
+            cart
+            for cart in carts
+            if (cart['status'] != 'deleted' and
```

Comment:
`cart['status'] not in ['deleted', 'disabled']`

Reply:
Changed.